setup.sh

```diff
@@ -318,6 +330,14 @@ function synchronize_files()
   warning " -> For a better experience with kw, please, open a new terminal."
 }
 
+function append_bashcompletion()
+{
+  local shellrc="$1"
```

Comment:
Add one extra line to separate the variable declaration from the rest of the code.